test/throw-from-number.test.js

```diff
@@ -0,0 +1,39 @@
+import getThrowFromNumber from '../src/throw-from-number.js';
+
+const test = QUnit.test;
+
+test('get-throw', (assert) => {
```

Comment:
These tests look really good! Once again, I would just recommend changing the name of the tests to reflect what specific case you're testing for ('a throw of 1 returns paper')